### DISCUSSION TITLE
@umijs/openapi 没有仓库吗？

### DISCUSSION THREAD

**Discussion by lcysgsg** [umijs/umi#6646] [General]
只有 `plugin-openapi` .

挺好用的，但是生成的代码是 `@umijs/openapi` 进行的，有问题反馈不了也没法改

**Comment by ningliaoyuan** [comment]
+1 关注

**Comment by ningliaoyuan** [comment]
好像是这个： https://github.com/chenshuai2144/openapi2typescript
